app/containers/ErrorBoundary/index.js

```diff
@@ -0,0 +1,54 @@
+import logger from 'electron-timber';
+import React from 'react';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import * as ToastActions from '../../actions/toasts';
+
+const initialState = { hasError: false };
+
+class ErrorBoundary extends React.Component {
+  state = initialState;
+
+  static getDerivedStateFromError() {
+    return { hasError: true };
+  }
+
+  componentDidCatch(error) {
+    logger.error(error);
+    this.props.actions.error('error', 'Something wrong happen');
```

Comment:
happen**ed**

Reply:
done ! sorry for this i am french and we are terrible in foreign language :)